R/dbdm_prob_weight.R

```diff
@@ -0,0 +1,165 @@
+#'  \item{\code{"opt1hval"}}{Possible (with opt1hprob probability) outcome of option 1}
+#'  \item{\code{"opt1lval"}}{Possible (with (1-opt1hprob) probability) outcome of option 1}
+#'  \item{\code{"opt2hval"}}{Possible (with opt2hprob probability) outcome of option 2}
+#'  \item{\code{"opt2lval"}}{Possible (with (1-opt2hprob) probability) outcome of option 2}
+#'  \item{\code{"choice"}}{If option1 was taken, choice == 1, else choice == 2.}
+#' }
+#' \strong{*}Note: The data.txt file may contain other columns of data (e.g. "Reaction_Time", "trial_number", etc.), but only the data with the column
+#' names listed above will be used for analysis/modeling. As long as the columns above are present and labelled correctly,
+#' there is no need to remove other miscellaneous data columns.
+#'
+#' \strong{nwarmup} is a numerical value that specifies how many MCMC samples should not be stored upon the
+#' beginning of each chain. For those familiar with Bayesian methods, this value is equivalent to a burn-in sample.
+#' Due to the nature of MCMC sampling, initial values (where the sampling chain begins) can have a heavy influence
+#' on the generated posterior distributions. The \code{nwarmup} argument can be set to a high number in order to curb the
+#' effects that initial values have on the resulting posteriors.
+#'
+#' \strong{nchain} is a numerical value that specifies how many chains (i.e. independent sampling sequences) should be
+#' used to draw samples from the posterior distribution. Since the posteriors are generated from a sampling
+#' process, it is good practice to run multiple chains to ensure that a representative posterior is attained. When
+#' sampling is completed, the multiple chains may be checked for convergence with the \code{plot(myModel, type = "trace")}
+#' command. The chains should resemble a "furry caterpillar".
+#'
+#' \strong{nthin} is a numerical value that specifies the "skipping" behavior of the MCMC samples being chosen
+#' to generate the posterior distributions. By default, \code{nthin} is equal to 1, hence every sample is used to
+#' generate the posterior.
+#'
+#' \strong{Contol Parameters:} adapt_delta, stepsize, and max_treedepth are advanced options that give the user more control
+#' over Stan's MCMC sampler. The Stan creators recommend that only advanced users change the default values, as alterations
+#' can profoundly change the sampler's behavior. Refer to Hoffman & Gelman (2014, Journal of Machine Learning Research) for
+#' more information on the functioning of the sampler control parameters. One can also refer to section 58.2 of the
+#' \href{http://mc-stan.org/documentation/}{Stan User's Manual} for a less technical description of these arguments.
+#'
+#' @references
+#' Jessup, R. K., Bishara, A. J., & Busemeyer, J. R. (2008). Feedback produces divergence from prospect theory in descriptive choice.
```

Comment:
Is this a proper citation? On line 7, there are three other papers that are not referenced here.